### DISCUSSION TITLE
Enable environments for private repositories without Github Enterprise subscription

### DISCUSSION THREAD

**Discussion by francos** [community/community#16952] [Actions]
Right now environments are only available for public repositories and private ones with a Github Enterprise subscription.

For small teams as ours who pay for a Team subscription (and for who paying for Github Enterprise is out of budget) it's quite limiting to not have this feature as it means we can't securely create a deployment Workflow on Github for our production environment.

It'd be ideal if this feature can be enabled for all paying subscriptions as it's a basic need for any project small or large.

**Comment by matteocontrini** [comment]
I found this discussion coming from Google thinking this limitation was still in place, but it turns out that shortly after this discussion was created environments were made available to the Team plan too:

>Therefore, today, we are excited to announce that private repositories in Team and Pro plans now have access to GitHub Actions environments, deployment branches, and environment secrets everywhere—in the past, only teams on the Enterprise plan had access to all these features with private repositories

https://github.blog/news-insights/product-news/github-pages-now-uses-actions-by-default/

Some features are still limited to Enterprise though:

>If you are on a GitHub Free, GitHub Pro, or GitHub Team plan, other deployment protection rules, such as a wait timer or required reviewers, are only available for public repositories.

https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-deployments/managing-environments-for-deployment
